utils/misc/misc.go

```diff
@@ -65,24 +81,29 @@ func GetRudderEventVal(key string, rudderEvent interface{}) (interface{}, bool)
 
 //ParseRudderEventBatch looks for the batch structure inside event
 func ParseRudderEventBatch(eventPayload json.RawMessage) ([]interface{}, bool) {
+	// logger.Debug("=== ParseRudderEventBatch === ")
 	var eventListJSON map[string]interface{}
 	err := json.Unmarshal(eventPayload, &eventListJSON)
 	if err != nil {
+		logger.Debug("==err== ", err)
```

Comment:
Make the log more detailed. 
logger.Debug("Parse Rudder Event batch: ", err)

Reply:
done